lib/python/cc_v3.1_testing.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+""" This script was done to quickly generate a summary of the current state of
+success and failure of tests on netcdf files with the latest version of the IOOS
+compliance checker v3.1 . This script can be used as part of a bash loop to test
+NetCDF files and create a CSV output such as:
+for f in `find . -type f -iname '*.nc'`; do ./cc_v3.1_testing.py $f >> checker3_result.csv ; done
+
+The current selection of files is available on AWS10, under $WIP_DIR/checker
+"""
+
+import re
+import sys
+
+from util import pass_netcdf_checker
+from netCDF4 import Dataset
+
+
+def list_nc_checks_to_perform(netcdf_file_path):
```

Comment:
This should always return 'cf:1.6' and 'imos:1.4' unless the `Conventions` attribute exists and lists 'IMOS-1.3'.
In that case it should return 'cf:1.6' and 'imos:1.3'.